beets/autotag/__init__.py

```diff
@@ -52,20 +52,25 @@ def apply_item_metadata(item, track_info):
     if track_info.data_source:
         item.data_source = track_info.data_source
 
-    if track_info.lyricist is not None:
-        item.lyricist = track_info.lyricist
-    if track_info.composer is not None:
-        item.composer = track_info.composer
-    if track_info.composer_sort is not None:
-        item.composer_sort = track_info.composer_sort
-    if track_info.arranger is not None:
-        item.arranger = track_info.arranger
-    if track_info.work is not None:
-        item.work = track_info.work
-    if track_info.mb_workid is not None:
-        item.mb_workid = track_info.mb_workid
-    if track_info.work_disambig is not None:
-        item.work_disambig = track_info.work_disambig
+    misc_fields = ['artist_id',
```

Comment:
Constants like this should be hoisted out to the module level and given descriptive constant-like (i.e., all caps) names. (I know that we had an old `misc_fields` value that disobeyed this dictum, but there's no time like the present to fix that.)